src/index.html

```diff
@@ -10,7 +11,38 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" data-qa="big">
+      <label>
+        <div class="big-bar search-bar">
+          <div class="big-bar__icon icon-search"></div>
+          <input
+            type="text"
+            name="bir-search-bar"
+            placeholder="Try &quot;Los Angeles&quot;"
+            class="big-bar__input search-input"
+          >
+        </div>
```

Comment:
redundant div wrappers
instead div consider to make icon as background for pseudo-element on label